tests/layer_tests/mo_python_api_tests/test_mo_convert_pytorch.py

```diff
@@ -595,6 +637,39 @@ def forward(self, x):
     return net, ref_model, {}
 
 
+def create_pytorch_module_with_optional_inputs_case1(tmp_dir):
+    net = make_pt_model_with_optional_input()
+    example_input = {"x": torch.zeros((1,3,10,10)), "y": torch.ones((1,3,10,10))}
+    ref_model = make_ref_pt_model_with_optional_inputs([-1, -1, -1, -1])
```

Comment:
Very interesting that `example_input` does not set input shapes of the generated model.
Is it planned so? May we have confusion from users? I think it is better to describe it in `help`.
Now it says `Sample of model input in original framework.`

Reply:
no, we do not plan restrict input shapes by example_inputs, we use it only for rank prediction. it is needed only for case when reverse infer can not properly setup it for improving user experience

Reply:
I think we need to describe this better in `help` because now we may have some misalignment among FEs.
Just to set rules for all FEs.

@popovaan, do we set `example_input` shapes for model conversion in TF FE or resulted input shapes can be different from `example_input` shapes?